lib/msun/src/e_pow.c

```diff
@@ -299,7 +299,9 @@ pow(double x, double y)
 	r  = (z*t1)/(t1-two)-(w+z*w);
 	z  = one-(r-z);
 	GET_HIGH_WORD(j,z);
-	j += (n<<20);
+    /* sign bit of z is 0.
+       sign bit of j will indicate sign of 0x3ff-biased exponent */
```

Comment:
Style(9) wants this comment to look more like
```
/*
 * sign bit of z is 0
 * sign bit of j will indicate sign of 0x3ff-biased exponent.
 */
```

Reply:
Ah, thanks. There was a mix of style in this file and i missed that this is preferred.